.github/workflows/lint.yml

```
@@ -16,12 +16,12 @@ jobs:
       with:
         python-version: '3.x'
     - name: Install black and pylint
-      run: pip install black~=22.3 pylint~=2.13,!=2.13.6
+      run: pip install black>=24.4.2 pylint>=3.2.3
```

```suggestion
      run: pip install black pylint
```
May as well just do this.